docs/_include/variable_hydro.rst

```diff
@@ -0,0 +1,134 @@
+
+    If only one H5 file is used to initialize a body object, variable hydrodynamics
+    will not be used, regardless of the ``option`` flag below.
+
+2. Enable Variable Hydrodynamics
+    Set the ``body.variableHydro.option`` flag to enable variable hydrodynamics:
+
+        ``body(1).variableHydro.option = 1;``
+
+    If the ``option`` flag is not turned on (``option==0``), then extra H5 files are ignored.
+
+3. Set the Initial Hydrodynamic Index
+    The flag ``body.variableHydro.hydroForceIndexInitial`` allows the user to set the
+    default hydrodynamic dataset. The initial set of hydrodynamic coefficients
+    represents the body at the start of a simulation and likely its equilibrium 
+    position. It is used to define the body's mass, center of gravity, 
+    and center of buoyancy. 
+
+        ``body(1).variableHydro.hydroForceIndexInitial = 1;``
+    
+    This parameter is flexible because an initial index of one is not always convenient
+    and may complicate indexing logic. For example, consider a flap-based device with
+    multiple hydrodynamic datasets at various pitch angles (-50:10:50). It is most convenient
+    and straightforward to treat these angles in numerical order in BEM simulations, 
+    indexing logic, and other data processing. In this case the initial position is at a pitch angle of 0 so 
+    ``body.variableHydro.hydroForceIndexInitial = 6;``.
+
+4. Control the varying hydrodynamics in Simulink
+    The index ``body1_hydroForceIndex`` in Simulink
+    (or ``body1_hydroForceIndex`` for body 2, etc) controls the hydrodynamic coefficients used
+    in the hydrodynamic force calculations at a given time.
+    Users *must* create their own logic and indexing signal in Simulink. A ``GoTo`` 
+    block named ``body1_hydroForceIndex``, ``body2_hydroForceIndex``, etc must take 
+    a signal for each variable hydro body so that the corresponding body block can select the correct 
+    hydrodynamic coefficients during the simulation
+
+    This example, from the Varible Hydro Passive Yaw application, takes in position, wave direction, and 
+    BEM directions, calculates the index at a given time, and sends it to a ``GoTo`` block named 
+    ``body1_hydroForceIndex``.
+
+    .. figure:: /_static/images/variable_hydro_logic_example.png
+        :width: 500pt
+        :figwidth: 500pt
+        :align: center
+
+.. Note::
+    Variable hydrodynamics is not compatible with the following features:
+
+    * State-space radiation calculations
+    * FIR Filter radiation calculations
+    * Generalized body modes
+    * Non-hydrodynamic and drag bodies
+    * Conditions that require a variable mass, center of gravity, or center of buoyancy
+
+Application
+""""""""""""
+See the :ref:`user-applications-variable-hydro` WEC-Sim_Application for a demonstration of setting up and using variable hydrodynamics.
+
+Additional Considerations
+""""""""""""""""""""""""""
```

Comment:
I still don't love this section here, but I see its value and I'm okay with it for now. We should consider if this is how we want to do a feature disclaimers, or whether they belong with the specific application.